packages/app-check-types/index.d.ts

```diff
@@ -28,17 +28,35 @@ export interface FirebaseAppCheck {
     siteKeyOrProvider: string | AppCheckProvider,
     isTokenAutoRefreshEnabled?: boolean
   ): void;
+
+  /**
+   *
+   * @param isTokenAutoRefreshEnabled - If set to true, enables SDK to
+   * automatically refresh AppCheck token as needed. If false, disables this
+   * functionality. This overrides any value set during `activate()`.
+   */
   setTokenAutoRefreshEnabled(isTokenAutoRefreshEnabled: boolean): void;
 }
 
+/**
+ * An AppCheck provider. This may be either the built-in reCAPTCHA provider
+ * or a custom provider. For more on custom providers, see
+ * https://firebase.google.com/docs/app-check/web-custom-provider
```

Comment:
```suggestion
 * An App Check provider. This can be either the built-in reCAPTCHA provider
 * or a custom provider. For more on custom providers, see
 * https://firebase.google.com/docs/app-check/web-custom-provider
```